crates/beerus-cli/src/runner.rs

```diff
@@ -301,19 +301,19 @@ pub async fn run(beerus: BeerusLightClient, cli: Cli) -> Result<CommandResponse>
 
             StarkNetSubCommands::AddDeclareTransaction {
                 max_fee,
-                version,
```

Comment:
"version" argument throughout was removed, did not see it used even in legacy data types so I removed it throughout this PR.

Reply:
While it looks like `BroadcastedDeclareTransactionV2` does remove version I think we need to leave version almost everywhere else as it is defined in the transaction type:
https://docs.starknet.io/documentation/architecture_and_concepts/Blocks/transactions/#declare_v2

Reply:
The language in the version description of the field seemed like it was on the way out and could be removed safely. I can put it back if it's not ready to be removed.